web/client/plugins/ResourcesCatalog/utils/ResourcesUtils.js

```diff
@@ -104,7 +105,7 @@ export const computePendingChanges = (initialResource, resource, resourceData) =
     ];
     const categoryOptions = {
         'thumbnail': {
-            tail: '/raw?decode=datauri',
+            tail: `/raw?decode=datauri&v=${uuid()}`,
```

Comment:
Why here using uuid? a sort of cache?
If so, let's specify in a comment the reason.